app/tests/spec/views/change_password.js

```diff
@@ -222,7 +222,14 @@ function (chai, _, $, sinon, AuthErrors, FxaClient, Metrics, p,
               assert.isTrue(view.fxaClient.changePassword.calledWith(
                   EMAIL, oldPassword, newPassword));
               assert.isTrue(view.fxaClient.signIn.calledWith(
-                      EMAIL, newPassword, relier));
+                  EMAIL,
+                  newPassword,
+                  relier,
+                  {
+                    reason: view.fxaClient.SIGNIN_REASON.PASSWORD_CHANGE,
+                    sessionTokenContext: account.get('sessionTokenContext')
```

Comment:
Why add `sessionTokenContext` to the args here?  IIUC it wasn't used in the previous version of this test.